tests/microos/verify_setup.pm

```diff
@@ -75,6 +76,7 @@ sub systemd_tests {
 }
 
 sub disk_tests {
+    return if is_opensuse;
```

Comment:
What's broken here?

Reply:
The test drive (3rd) is formatted as ext4.

Reply:
Needs a comment. There's also a mismatch between what the script checks for `grep -q tumbleweed /etc/os-release` vs. `is_opensuse`.